releasenotes/notes/bump-magnum-capi-helm-7e4ad37d3d9eecce.yaml

```diff
@@ -0,0 +1,4 @@
+---
+upgrade:
```

Comment:
Upgrade is for notes that the user should be aware of when upgrading i.e. breaking changes. This is more of a feature (or fix if the update is to fix an issue)